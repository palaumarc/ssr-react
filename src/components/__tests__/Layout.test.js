import React from 'react';
import Layout from '../Layout';
import renderer from 'react-test-renderer';

jest.mock("react-router-dom")

it('All props are defined. Snapshot', () => {
  const tree = renderer
    .create(<Layout />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});