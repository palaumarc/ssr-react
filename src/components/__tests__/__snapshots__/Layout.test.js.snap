// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All props are defined. Snapshot 1`] = `
<div>
  <switch-mock>
    <route-mock
      component={[Function]}
      exact={true}
      path="/"
    />
    <route-mock
      component={[Function]}
      exact={true}
      path="/games/:id"
    />
  </switch-mock>
</div>
`;
